scene/3d/physics_body_3d.cpp

```diff
@@ -3251,6 +3259,18 @@ Vector3 PhysicalBone3D::get_joint_rotation() const {
 	return joint_offset.basis.get_euler_normalized();
 }
 
+void PhysicalBone3D::set_joint_exclude_nodes_from_collision(bool p_exclude_nodes_from_collision) {
+    if (joint_exclude_nodes_from_collision == p_exclude_nodes_from_collision) {
```

Comment:
Indentation uses tabs, you can fix this by using clang, see [here](https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html)

Reply:
Done! 

PS. Sorry for making some dumb mistakes. This is my first time contributing here

Reply:
It's a learning experience! That's why we are here to help